crates/holochain/src/conductor/conductor/agent_key_operations.rs

```diff
@@ -1,171 +0,0 @@
-//! Tests related to key revocation are located under [tests/agent_key_revocation](tests).
```

Comment:
I've removed the call `RevokeAgentKey` too. It was functional without DPKI, but I don't think it makes sense to keep it, as there's no way to update an agent key. I can easily revert this if desired.